auslib/test/web/test_client.py

```diff
@@ -805,6 +805,23 @@ def testAvastURLsWithGoodQueryArgs(self):
 </updates>
 """)
 
+    @given(text(alphabet=characters(blacklist_categories=('Cs', 'Po')), max_size=128))
```

Comment:
What's the reason to blacklist some punctuation categories but not others?

Reply:
I suggested ignore the punctuation/other unicode category (Po), because it can insert caracters that connexion assumes as extra parameters that we not have in spec. Ignore "Cs" category is a default behavior of text() when alphabet is none.

Reply:
Got it - thanks for the explanation.